chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1654,7 +1545,7 @@ impl Handler<SetChainInfo> for PeerManagerActor {
         if !state.accounts_data.set_keys(info.tier1_accounts.clone()) {
             return;
         }
-        tokio::spawn(async move {
+        ctx.spawn(wrap_future(async move {
```

Comment:
Is this to keep the thing on the current thread out of caution, or is there a deeper reason?